index.js

```diff
@@ -214,13 +214,16 @@ var re = new RegExp(
     // $2 = function name
     // $3 = method name
   '(?:([^\\(\\[]*)(?: \\[as ([^\\]]+)\\])? \\()?' +
+    // Object.[Symbol.…] (, maybe
+    // $4 = function name
+  '(?:([^\\(\\[]+\\[Symbol\\..+?\\]) \\()?' +
```

Comment:
Oh, actually there's all kinds of madness that can go in there.

Ok, I'm going to land this PR and then fix it with some more thorough tests.  Stand by.